test/index.js

```diff
@@ -49,6 +49,32 @@ describe('strange-forms', () => {
             done();
         });
 
+        it('does not update a prop in state when it hasn\'t changed.', (done) => {
+
+            const props = {
+                name: 'MVX',
+                age: 13
+            };
+
+            const Component = class extends StrangeSansLifecycle {
+
+                constructor(initProps) {
+
+                    super(initProps);
+                    this.strangeForm({ get: null, fields: ['name', 'age'] });
+                }
+            };
+
+            const component = new Component(props);
+
```

Comment:
The simplest fix for this test would be to insert something on this line like,
```js
            component.state._sf_name = 'JRD'; // Invalid
```

Or to go a step towards integration, actually using `proposeNew()` to get into that `state`.